### PR TITLE
Added an alert for form validation

### PR DESCRIPTION
**Pull request by rshin8763**

### CODE REVIEW

File: views/queue-manager.hbs [hunk 1]
```diff
@@ -52,16 +52,22 @@
 
 <h5>Edit Priority:</h4>
 <form>
-  <div class="form-row">
-    <div class="form-group col-md-4">
-      Patient ID &nbsp
-      <input type="number" min="0" step="1" name="pid" id="pid"><br>
+    <div class="form-row">
+        <div class="form-group col-md-4">
+            Patient ID &nbsp
```
Comment (tymc7) [inline views/queue-manager.hbs]: should there be a `;` after the `&nbsp`?

File: views/queue-manager.hbs [hunk 2]
```diff
@@ -52,16 +52,22 @@
...
+            <input type="number" min="0" step="1" name="pid" id="pid"><br>
+        </div>
+        <div class="form-group col-md-4">
+            New Priority &nbsp
```
Comment (tymc7) [inline views/queue-manager.hbs]: Same here

**Review by tymc7 — COMMENTED** [review]
Also, this is more for discussion, but why choose non-breaking spaces rather than a `form-inline` class?

**Review by ethanysc — COMMENTED** [review]
Merging so you can test.

**Review by ethanysc — APPROVED** [review]

**Pull request merged**